src/utils/query.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * 쿼리 파라미터를 받아 항상 문자열 배열로 반환합니다.
+ */
+export const getQueryParam = (
+  param: string | string[] | undefined,
+): string[] => {
+  if (Array.isArray(param)) {
+    return param;
+  }
+  if (param === undefined || param === '') {
+    return [];
+  }
+  return [param];
+};
```

Comment:
```suggestion
export const getQueryParams = (
  params: string | string[] | undefined,
): string[] => {
  if (Array.isArray(param)) {
    return params;
  }
  
  if (params === undefined || params === '') {
    return [];
  }
  
  return [params];
};
```

p3;
- 네이밍 수정 : getQueryParam > getQueryParams
params를 전달받고 문자열 배열을 반환하므로 복수형이 조금 더 적절할 것 같습니다!
- 조건문 또는 return문 위에 개행 한 번씩 넣어주시면 가독성 좋아질 것 같아요!

좋은 유틸함수 생성해주셔서 감사해요!